### ISSUE TITLE
Does zaw support Shift+Tab?

### ISSUE THREAD

**Issue by moonrumble** [zsh-users/zaw#82]
I started using zaw since last 2 days and I am loving it. I noticed when playing around if I press Tab or Ctrl+I in the zaw-history. I can't go back.

**Comment by willghatch** [comment]
Not really.  I think it would be somewhat difficult to go back (granted - I'm not the original author, and I haven't even done small upgrades for quite a while.  Without going and digging through the code again I can't say for certain if it would be hard, but my impression from working on it occasionally is that it would be).  But if you want to hack at it and make a PR, that would be fine.

**Comment by zizhengwu** [comment]
This would be extremely useful. Because sometimes I need to Tab to view a very long multi-line command. Unfortunately, there is no way to go back if I found it is not what I wanted.
